manifests/dashboard-install/dashboard-clusterrolebinding.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  labels:
+    app.kubernetes.io/component: rollouts-dashboard
+    app.kubernetes.io/name: rollouts-dashboard-clusterrolebinding
+    app.kubernetes.io/part-of: argo-rollouts
+  name: argo-rollouts-dashboard-clusterrolebinding
```

Comment:
How about removing the suffix, which seems redundant. There is an issue to track this: https://github.com/argoproj/argo-rollouts/issues/1231

Reply:
Do you mean `app.kubernetes.io/name` label should be `rollouts-dashboard`?
Or you're talking about the **name** field?

Reply:
The `name` field can be `argo-rollouts-dashboard`. I think this is applicable to the label too. Thanks.

Reply:
what @huikang meant is to keep as  argo-rollouts-dashboard and remove clusterrolebinding

Reply:
done